assets/js/components/ImgDropAndCrop/index.js

```diff
@@ -0,0 +1,139 @@
+import React, {Component} from 'react'
+import {Button} from 'reactstrap'
+import Dropzone from 'react-dropzone'
+/*eslint-disable */
+import ReactCrop from 'react-image-crop'
+/*eslint-enable */
+
+import {connect} from 'react-redux'
+import {FormattedMessage, injectIntl} from 'react-intl'
+import {image64toCanvasRef, extractImageFileExtensionFromBase64} from './ReusableUtils';
+
+const imageMaxSize = 1000000000000000
+const acceptedFileTypes = 'image/x-png, image/png, image/jpg, image/jpeg, image/gif'
+const acceptedFileTypesArray = acceptedFileTypes.split(',').map((item) => {return item.trim()})
+class ImgDropAndCrop extends Component {
+    constructor(props){
+        super(props)
+        this.imagePreviewCanvasRef = React.createRef()
+        this.state = {
+            imgSrc: null,
+            crop: {
+                aspect: 1 / 1,
+            },
+            pixelCrop: null,
+        }
+    }
+
+    verifyFile = (files) => {
+        if (files && files.length > 0) {
+            const currentFile = files[0]
+            const currentFileType = currentFile.type
+            const currentFileSize = currentFile.size
+            if(currentFileSize > imageMaxSize) {
+                alert(<FormattedMessage id="profile.fileSize.error" /> + currentFileSize + <FormattedMessage id="byte.size.error" />)
```

Comment:
You can't use FormattedMessage in alerts. Since you already inject intl you should instead use `this.props.intl.formatMessage` that returns a string.
I also think that the messages should be combined and pass the file-size as a value to that message.